regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -80,3 +80,9 @@ <h3 class="toc-type">Table of Contents</h3>
 {% endblock %}
 
 {% block chrome-sidebar %}{% endblock %}
+
+{% block preload %}
+<script>
+  window.DOC_NUMBER = '{{ doc_number }}';
+</script>
```

Comment:
We should avoid adding data to the global namespace where possible. Is there a way to have engineered this so that `window.DOC_NUMBER` isn't required?